### PR TITLE
Boards: Make detection systems hardware agnostic

### PR DESCRIPTION
**Pull request by tombettany**
Abstract board settings to make it simple to support boards with other
settings.

Requires: https://github.com/KanoComputing/kano-toolset/pull/193

First draft and not tested yet but what do you guys think of the approach?

@convolu @radujipa @Ealdwulf

### CODE REVIEW

File: kano_settings/system/boards/rpi_1.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    'Wifi-connection-attempted': False,
+    'Overclocking': 'High',
+    'Mouse': 'Normal',
+    'Font': 'Normal',
+    'Wallpaper': 'kanux-background',
+    'Parental-level': 0,
+    'Locale': 'en_US',
+    'LED-Speaker-anim': False,
+    'Use_GLX': False
+}
```
Comment (Ealdwulf) [inline kano_settings/system/boards/rpi_1.py]: Do we need all of these things in the board config? Eg, the wallpaper, font etc don't seem to be related to the board.

Comment (tombettany) [inline kano_settings/system/boards/rpi_1.py]: Fair point - it really is just the default overclock setting which is board specific.

Comment (Ealdwulf) [inline kano_settings/system/boards/rpi_1.py]: And LED-speaker-anim. But otherwise yes.

**Comment by Ealdwulf** [comment]
Apart from the comment above this looks like a great simplification :+1:

**Comment by tombettany** [comment]
@Ealdwulf fixed your comment above. I'll merge for us to test.
